include/z64pause.h

```diff
@@ -148,7 +158,7 @@ typedef struct PauseContext {
     /* 0x0160 */ char unk_160[0x04];
     /* 0x0164 */ Vtx* questVtx;
     /* 0x0168 */ Vtx* cursorVtx;
-    /* 0x016C */ Vtx* saveVtx;
+    /* 0x016C */ Vtx* promptPageVtx;
```

Comment:
I feel like we should say "save prompt" rather than "prompt" in most places (e.g. like PAUSE_SAVE_PROMPT_STATE_APPEARING, savePromptState, sSavePromptMessageTexs rather than R_KALEIDO_PROMPT_CURSOR_ALPHA, sKaleidoPromptCursorAlphaVals, KALEIDO_PROMPT_CURSOR_R, PromptQuad). Maybe there is only one prompt in kaleido but it doesn't immediately make me think of the save prompt

Reply:
it is the save and game over prompt. while both are asking you if you want to save, i think the name "save prompt" makes it easy to think its only for the pause menu use case

Reply:
Hm I think the game over prompt as also being the pause menu. But fair enough

Reply:
> the explicit yes/no question is

yeah that's what the "prompt" being referred to is, I only mention "press B" as the action for bringing up the (pause menu) save prompt

the game over prompt also has saving indeed but also continue yes/no

Reply:
I guess now it is more about PauseSavePromptState and savePromptState, since based on the name I thought it was used for both the save prompt in the pause menu and the save prompt in the game over screen, and was surprised that it wasn't when I read the code more closely. But that terminology already exists (e.g. PAUSE_STATE_SAVE_PROMPT) and I don't immediately have a better idea and don't want to hold up the PR. Maybe a comment somewhere?

Reply:
hmm I guess this will indeed be confusing in the future with gameover states more documented, for example `PAUSE_STATE_14` may be `PAUSE_STATE_GAMEOVER_SAVE_PROMPT`, it also uses `sSavePromptMessageTexs` ("Would you like to save?" in English)

I hope the "gameover" prefix will be enough

for now I added two comments 879616e54

Reply:
yeah maybe in the future "pause save prompt" and "game over save prompt" would be a good way to distinguish them